watertap/unit_models/zero_order/mabr_zo.py

```diff
@@ -37,14 +37,89 @@ def build(self):
         self._tech_type = "mabr"
 
         build_sido_reactive(self)
-        constant_intensity(self)
+
+        self.nitrogen_removal_rate = Var(
+            units=pyunits.g / pyunits.m**2 / pyunits.day,
+            bounds=(0, None),
+            doc="Nitrogen removal rate per day",
+        )
+
+        self._fixed_perf_vars.append(self.nitrogen_removal_rate)
 
         self.blower_size = Var(
             units=pyunits.m**2,
             bounds=(0, None),
             doc="Sizing variable for blower size",
         )
 
-        self._fixed_perf_vars.append(self.blower_size)
+        @self.Constraint(
+            self.flowsheet().time,
+            doc="Constraint for blower size",
+        )
+        def blower_size_consumption(b, t):
+            return b.blower_size == pyunits.convert(
+                b.properties_treated[t].flow_mass_comp["ammonium_as_nitrogen"]
```

Comment:
Since we are hard-coding this component within the constraint, you'll need to add a check above to ensure that the "ammonium_as_nitrogen" component is provided in the solute list. If not, raise an exception.

Reply:
@adam-a-a Good point, but since the `ammonium_as_nitrogen` is the `key_reactant`, so there is already a check in `build_sido_reactive` to ensure that is must be included.